Gemfile

```diff
@@ -27,6 +27,7 @@ gem "puppet", :path => File.dirname(__FILE__), :require => false
 gem "facter", *location_for(ENV['FACTER_LOCATION'] || ['> 2.0', '< 4'])
 gem "hiera", *location_for(ENV['HIERA_LOCATION'] || ['>= 2.0', '< 4'])
 gem "rake", "10.1.1", :require => false
+gem "msgpack", "~> 0.7.6", :require => false
```

Comment:
We'll need to resolve the issues we had with semantic_puppet before adding a hard dependency on msgpack, e.g. puppet-agent, puppetserver, Gemfile, ext/project_data.yaml, etc.